hadoop-plugin/src/main/groovy/com/linkedin/gradle/azkaban/AzkabanHelper.groovy

```diff
@@ -208,25 +204,58 @@ class AzkabanHelper {
   }
 
   /**
-   * Makes sure that the pre-requisites of using the azkaban tasks are satisfied.
+   * Configures the fields for azkabanUploadTask.
    *
    * @param azkProject The AzkabanProject
+   * @return boolean to save the entered fields to .azkabanPlugin.json
    */
-  static void configureTask(AzkabanProject azkProject) {
+  static boolean configureTask(AzkabanProject azkProject) {
     def console = System.console();
     if (console == null) {
       throw new GradleException(
           "\nCannot access the system console. To use this task, explicitly set JAVA_HOME to the version specified in product-spec.json (at LinkedIn) and pass --no-daemon in your command.");
     }
 
-    if (!azkProject.azkabanUserName) {
-      azkProject.azkabanUserName = console.readLine("\nAzkaban user name: ");
-      logger.lifecycle("Run writeAzkabanPluginJson task and enter azkabanUserName field in .azkabanPlugin.json to automatically fetch the username.");
-    }
+    println("INTERACTIVE MODE:");
```

Comment:
Use logger.lifecycle instead of println to ensure the message prints atomically instead of possibly being cut into pieces